CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 #### 0.10.0
 
 - Allows to declare a schema inside the yaml file through modeline `# yaml-language-server: $schema=<urlOfTheSchema>`
+- Insert empty string instead of 'null' for string array completion [#277](https://github.com/redhat-developer/yaml-language-server/pull/277)
+- support schema associations in file [#280](https://github.com/redhat-developer/yaml-language-server/pull/280)
```

Comment:
this one is the same than l.3 Allows to declare a schema inside the yaml file through modeline `# yaml-language-server: $schema=<urlOfTheSchema>`